packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -2,22 +2,40 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @ts-nocheck
```

Comment:
This affects the entire file right?  Kinda overdoing it!

Reply:
I don't know of any way to disable the check for just one part of the file :-\

Reply:
I made a couple suggestions below that I think will obviate the need for this.